### DISCUSSION TITLE
Getting invalid event source

### DISCUSSION THREAD

**Discussion by madanvparadkar** [Tracardi/tracardi#854] [General]
Though I have added the correct event source using the gui and used it in javascript on my website. I received the following error - ERROR:app.api.track.event_server_endpoint:Invalid event source 

The response to the track call does show the correct event_source but with an error 
"detail": "Invalid event source `046c1e19-4e7d-4da8-9d8a-98218f806df6`.

**Comment by madanvparadkar** [comment]
Added in Q&A section and deleted it from here

**Comment by atompie** [comment]
Is the event source published and enabled.
